composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/ResultWithNetworkAndAggregates.kt

```diff
@@ -3,7 +3,9 @@ package org.ooni.probe.data.models
 data class ResultWithNetworkAndAggregates(
     val result: ResultModel,
     val network: NetworkModel?,
-    val measurementsCount: Long,
+    val doneMeasurementsCount: Long,
+    val failedMeasurementsCount: Long,
+    val anomalyMeasurementsCount: Long,
```

Comment:
Maybe something like this.
`measurementCount: MeasurementCount`

`
data class MeasurementCount(
    val done: Int,
    val failed: Int,
    val anomaly: Int
) {
    fun total() = done + failed + anomaly
}
`